dev/applications/mixed-app-network/mixed-app-network.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: mixed-app
+spec:
+  components:
+    - name: cloud-worker-1
+      type: main-worker
+      properties:
+        image: nginx
+        name: simplenginx
+      traits:
+        - type: labels
+          properties:
+            "app": "cloud-worker-1"
+    - name: edge-worker-1
+      type: edge-worker
+      properties:
+        image: nginx
+        name: simplenginx
+      traits:
+        - type: labels
+          properties:
+            "app": "edge-worker-1"
+    - name: edge-worker-2
+      type: edge-worker
+      properties:
+        image: nginx
+        name: simplenginx
+      traits:
+        - type: labels
+          properties:
+            "app": "edge-worker-2"
+    - name: network
+      type: application-network
```

Comment:
didn't i need to add usersnames here as well? How to enable users secrets here?

Reply:
please update network to full example:
```yaml
    - name: network
      type: application-network
      properties:
        usernames:
          - cloud-worker-1
          - edge-worker-1
          - edge-worker-2
```

Reply:
Better idea,... remove this example here and use `dev/manifests/applications/mixed-app-network/mixed-app-network.yaml` from [edgefarm.applications](https://github.com/edgefarm/edgefarm.applications) instead.

Reply:
or use some more of these examples?

Reply:
using dev/manifests/applications/mixed-app/mixed-app-network.yaml